tools/yulPhaser/Chromosome.cpp

```diff
@@ -0,0 +1,148 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <tools/yulPhaser/Chromosome.h>
+
+#include <tools/yulPhaser/Random.h>
+
+#include <libyul/optimiser/BlockFlattener.h>
+#include <libyul/optimiser/CommonSubexpressionEliminator.h>
```

Comment:
Do we need all these includes?

Reply:
Right, we don't. I only needed them for debugging. I'll remove them.

Reply:
Done.